setup.cfg

```diff
@@ -1,3 +1,50 @@
+[metadata]
+name = aiohttp-cors
+version = 0.7.0
```

Comment:
version should be taken from `about.py`

Reply:
It is, just forgot to remove from the config.

https://github.com/aio-libs/aiohttp-cors/pull/212/files#diff-2eeaed663bd0d25b7e608891384b7298R35